### ISSUE TITLE
Add "fixed" elements on the side of rows

### ISSUE THREAD

**Issue by marciogurka** [bryntum/support#5152]
[Forum post](https://www.bryntum.com/forum/viewtopic.php?f=51&t=22211&p=109844#p109844)

Problem description:
Really often, it's quite hard to read a Gantt / scheduler when starting from a "blank" part of the agenda, or when navigating in a blank part of a Gantt.

For that, we'd like to:

- Show the name of the task as "fixed" on the left

- Show right/left arrows to scroll to the task horizontally

In the attachment is a video of what we'd like to do with Bryntum.

https://user-images.githubusercontent.com/16693227/187741976-f56dea7c-d9b3-4f27-b81c-816e9c7939d2.mov

**Comment by Aarbel** [comment]
@marciogurka 

additional example from Notion's gantts

<img width="796" alt="image" src="https://user-images.githubusercontent.com/25119847/188402742-226d6085-d11a-4bdf-a37a-b0049b748d69.png">

https://user-images.githubusercontent.com/25119847/188403089-c16d9c18-7d64-4255-90bf-87c003ecce00.mov

**Comment by Aarbel** [comment]
@marciogurka do you plan to provide it in V6 ?

**Comment by marciogurka** [comment]
Hey @Aarbel it's still in discussion, so far no new updates on this.

**Comment by marciogurka** [comment]
[+1](https://www.bryntum.com/forum/viewtopic.php?p=112617#p112617) for this one

**Comment by Aarbel** [comment]
@marciogurka do you have any news about it ? 

Or any way to add it with custom code on the gantt / scheduler ?

**Comment by matsbryntse** [comment]
We should be able to get this part of a release in Q3!

**Comment by matsbryntse** [comment]
https://github.com/bryntum/support/assets/218570/387b893c-e5e1-4c51-b670-d8d5c2af7fcb

**Comment by Aarbel** [comment]
Great news, thanks a lot @matsbryntse Are there options to put the row arrows to the left or the right function of bar placement ?

**Comment by Aarbel** [comment]
Great news, thanks a lot @matsbryntse 

Few ideas to make sure the feature will be fully usable by many teams:

Being able to :
- customise the arrow with a React component
- show datas next to the arrow (like the start / end date) can be a good
- being able to know if separately the start date is hidden (off the scroll), the end date is hidden, the whole bar is hidden (see screenshot of Notion's timeline view bellow)


https://github.com/bryntum/support/assets/25119847/f9455cd6-e90e-427a-895b-2df525a1aaa0

**Comment by Aarbel** [comment]
@SergeyMaltsev 🎉

**Comment by Aarbel** [comment]
@SergeyMaltsev when do you plan to release bryntum 5.6 ?

**Comment by SergeyMaltsev** [comment]
@Aarbel You may use released 5.6.0-beta-1 version with this fix.
5.6.0 release is planned pretty soon (in a week or two).

**Comment by Aarbel** [comment]
@SergeyMaltsev thanks a lot for the details, 

Is this enhancement included inside Bryntum Scheduler 5.6.0 and Bryntum Gantt 5.6.0 ?

**Comment by matsbryntse** [comment]
It's only for Bryntum Gantt for now, would you need something similar for Scheduler?

**Comment by Aarbel** [comment]
@matsbryntse yes of course, it's the same pain for users (jumping horizontally if elements are not visible on the right or the left)

**Comment by Aarbel** [comment]
@marciogurka i'm not seeing any issue about this, could be really great to include it very soon in the scheduler !
